tests/APIs/ConsultationApiTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace EscolaLms\Consultations\Tests\APIs;
+
+use EscolaLms\Consultations\Database\Seeders\ConsultationsPermissionSeeder;
+use EscolaLms\Consultations\Models\Consultation;
+use EscolaLms\Consultations\Tests\TestCase;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Testing\Fluent\AssertableJson;
+
+class ConsultationApiTest extends TestCase
+{
+    use DatabaseTransactions;
+    private Consultation $consultation;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->seed(ConsultationsPermissionSeeder::class);
+
+        $this->user = config('auth.providers.users.model')::factory()->create();
+        $this->user->guard_name = 'api';
+        $this->user->assignRole('tutor');
+        $this->consultation = Consultation::factory()->create();
+        $this->consultation->author()->associate($this->user);
+    }
+
+    public function testConsultationsList()
+    {
+        $this->response = $this->actingAs($this->user, 'api')->get('/api/admin/consultations');
+        $this->response->assertOk();
```

Comment:
Assert that correct data is returned, because this test would pass even if you always returned empty 200 response...